packages/jest-mock-web-client/README.md

```diff
@@ -0,0 +1,100 @@
+# Jest Mock Slack Web Client
+
+This package contains a type-safe mock of `@slack/web-api`.
+
+## Install
+
+```bash
+yarn add --dev @slack-wrench/jest-mock-web-client
+# or
+npm install --save-dev @slack-wrench/jest-mock-web-client
+```
+
+## Usage
+
+This mock is intended to be used as a [manual mock](https://jestjs.io/docs/en/manual-mocks#mocking-node-modules).
+
+Create a file for the web-api manual mock (`__mocks__/@slack/web-api.js`)
+
+```js
+// __mocks__/@slack/web-api.js
+const mockWebApi = require('@slack-wrench/jest-mock-web-client');
+
+module.exports = mockWebApi(jest);
+```
+
+Test your code how you would expect:
+
+```js
+// src/bot.spec.js
+const { WebClient } = require('@slack/web-api');
+const Bot = require('./bot');
+
+describe('The Bot', () => {
+  let bot;
+  let client;
+
+  beforeEach(() => {
+    // Reset mocks for each test
+    WebClient.clearMocks();
+    // In this example, bot create's a `new WebClient`
+    bot = new Bot();
+    client = WebClient.mock.instances[0];
+  });
+
+  it('Can say things', async () => {
+    const message = 'Mocking Slack is easy!';
+    await bot.say(message);
+
+    expect(client.chat.postMessage).toHaveBeenCalledWith(
+      expect.objectContaining({
+        text: message,
+      }),
+    );
+  });
+});
+```
+
+### Getting the mocked client
+
+Mocked instances are exposed in the normal way for [automatic mocks](https://jestjs.io/docs/en/es6-class-mocks#automatic-mock). You can find created instances at `WebClient.mock.instances`.
```

Comment:
I don't know that we should assume when people are reading this that they understanding what you mean by "the normal way". This might be clearer as just
```suggestion
Mocked instances are exposed as [jest automatic mocks](https://jestjs.io/docs/en/es6-class-mocks#automatic-mock). You can find created instances at `WebClient.mock.instances`.
```
?